src/storage/query/QueryBaseProcessor-inl.h

```diff
@@ -152,6 +152,13 @@ nebula::cpp2::ErrorCode QueryBaseProcessor<REQ, RESP>::buildFilter(
     if (filter_ == nullptr) {
       return nebula::cpp2::ErrorCode::E_INVALID_FILTER;
     }
+    const auto* vertexFilterStr = getFilter(req, true);
```

Comment:
Be careful here, if the filter is vertex-only, both filter_ and vertexFilter_ will be set, the Filter will eval more than once?

Reply:
Yes, it will be executed twice if vertexFilter_ and filter_ are the same And the result is true

Reply:
Is `filter` and `vertex-filter` mutually exclusive? (can't set them both at the same request)? If is, we only need to eval once I think~

BTW, where do we extract vertex filter? I didn't see the code to set it.